### PR TITLE
form validation

### PR DESCRIPTION
**Pull request by peteronyam**
This pull request updates the Readme file, and styles.css, and also adds a javascript file for contact form validation to enhance the project's readability and provide more comprehensive information about the project's purpose and goals. The changes aim to help new contributors understand the project and how to contribute to it by adding installation instructions, troubleshooting steps, and a step-by-step guide for building and running the project.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+});
+
+const form = document.querySelector('form');
+const emailInput = document.querySelector('#email');
+
+form.addEventListener('submit', (event) => {
+  if (emailInput.value !== emailInput.value.toLowerCase()) {
+    event.preventDefault();
+    const error = document.createElement('p');
+    error.textContent = 'Please enter your email in lower case.';
```
Comment (Munsa1) [inline main.js:43]: - [ ] The error message should show up close to the submit button or the input fields. (i.e error message can show up around the input fields or just around the submit button). Kindly fix this issue.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  } else {
+    header.classList.remove('header-fixed');
+  }
+});
+
+const form = document.querySelector('form');
+const emailInput = document.querySelector('#email');
+
+form.addEventListener('submit', (event) => {
+  if (emailInput.value !== emailInput.value.toLowerCase()) {
```
Comment (Munsa1) [inline main.js:40]: - [ ] Great job implementing the eventlistener, however in your HTML you do not have a form with the classname `form` this causes your code to not execute. kindly fix this

Comment (Munsa1) [inline main.js:40]: - [ ] Great job implementing the eventlistener, however in your HTML you do not have a form with the classname `form` this causes your code to not execute. kindly fix this  

* If a user submits the contact form with the email field containing uppercase characters an error message should show up.
* If a user submits the contact form with the email field containing only lowercase characters the form is sent normally.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @peteronyam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Setup Passing Linters :heavy_check_mark: 
Followed JavaScript Best Practices :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._
- [ ] Check the inline comments below


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**